projects/admin/src/app/app.module.ts

```diff
@@ -15,70 +15,74 @@
-import { ItemTypeDetailViewComponent } from './record/detail-view/item-type-detail-view.component';
-import { LibraryDetailViewComponent } from './record/detail-view/library-detail-view/library-detail-view.component';
-import { DayOpeningHoursComponent } from './record/detail-view/library-detail-view/day-opening-hours/day-opening-hours.component';
-import { ExceptionDateComponent } from './record/detail-view/library-detail-view/exception-date/exception-date.component';
-import { DocumentDetailViewComponent } from './record/detail-view/document-detail-view/document-detail-view.component';
-import { HoldingComponent } from './record/detail-view/document-detail-view/holding/holding.component';
-import { HoldingItemComponent } from './record/detail-view/document-detail-view/holding-item/holding-item.component';
-import { HoldingsComponent } from './record/detail-view/document-detail-view/holdings/holdings.component';
-import { CircPolicyDetailViewComponent } from './record/detail-view/circ-policy-detail-view/circ-policy-detail-view.component';
-import { LocationDetailViewComponent } from './record/detail-view/location-detail-view/location-detail-view.component';
-import { LocationComponent } from './record/brief-view/libraries-brief-view/location/location.component';
-import { ItemDetailViewComponent } from './record/detail-view/item-detail-view/item-detail-view.component';
-import { ItemAvailabilityComponent } from './record/item-availability/item-availability.component';
-import { ItemTransactionComponent } from './record/detail-view/item-detail-view/item-transaction/item-transaction.component';
-import { ItemTransactionsComponent } from './record/detail-view/item-detail-view/item-transactions/item-transactions.component';
-import { AuthorNameTranslatePipe } from './pipe/author-name-translate.pipe';
-import { PatronDetailViewComponent } from './record/detail-view/patron-detail-view/patron-detail-view.component';
-import { RefComponent } from './record/editor/ref/ref.component';
-import { RemoteAutocompleteInputTypeComponent } from './record/editor/remote-autocomplete/remote-autocomplete.component';
-import { NoCacheHeaderInterceptor } from './interceptor/no-cache-header.interceptor';
-import { InterfaceInfoComponent } from './interface-info/interface-info.component';
-import { DocumentEditorComponent } from './record/custom-editor/document-editor/document-editor.component';
-import { VendorDetailViewComponent } from './record/detail-view/vendor-detail-view/vendor-detail-view.component';
-import { VendorBriefViewComponent } from './record/brief-view/vendor-brief-view.component';
-import { AddressTypeComponent } from './record/detail-view/address-type/address-type.component';
+  BsDatepickerModule,
+  BsLocaleService,
+  CollapseModule,
+  TabsModule,
+  TooltipModule,
+  TypeaheadModule
+} from 'ngx-bootstrap';
+import {FrontpageComponent} from './frontpage/frontpage.component';
```

Comment:
This becomes quite long. We probably need to split our application into several modules in a near future.

Reply:
ok